nuxeo-features/rest-api/nuxeo-rest-api-server/src/main/java/org/nuxeo/ecm/restapi/server/jaxrs/BatchUploadObject.java

```diff
@@ -116,6 +117,9 @@
 
     protected static final String REQUEST_HANDLER_NAME = "handlerName";
 
+    /** @since 11.2 */
```

Comment:
As this property does not exist in `11.2`, I would update the since for the next HF.

Reply:
aligned with 10.10-HF30